clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobWorkerImpl.java

```diff
@@ -214,10 +219,14 @@ private void backoff(final JobPoller jobPoller, final Throwable error) {
       LOG.warn(SUPPLY_RETRY_DELAY_FAILURE_MESSAGE, e);
       pollInterval = DEFAULT_BACKOFF_SUPPLIER.supplyRetryDelay(prevInterval);
     }
-    LOG.debug(
-        "Failed to activate jobs due to {}, delay retry for {} ms",
-        error.getMessage(),
-        pollInterval);
+    if (error != null) {
```

Comment:
:question: Did you observe this? We call `onPollError` without an actual error?

Reply:
I added this conditional because I thought of reusing this method for the stream backoff, and in there we call `backoff(jobPoller, null);`

Reply:
I would propose to refactor it for this new use case then, even if it's a little bit more changes.